patch/kernel/rockchip64-current/board-rk3328-roc-cc-dts-enable-dmc.patch

```diff
@@ -0,0 +1,92 @@
+diff --git a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
+index a1041ec3e..cc5855eda 100644
+--- a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
++++ b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
+@@ -4,6 +4,7 @@
+  */
+ 
+ /dts-v1/;
++#include "rk3328-dram-default-timing.dtsi"
+ #include "rk3328.dtsi"
+ 
+ / {
+@@ -14,6 +14,79 @@
+ 		stdout-path = "serial2:1500000n8";
+ 	};
+ 
++	dfi: dfi@ff790000 {
++		reg = <0x00 0xff790000 0x00 0x400>;
++		compatible = "rockchip,rk3328-dfi";
++		rockchip,grf = <&grf>;
++		status = "okey";
```

Comment:
status = "okay"

Reply:
I always do that mistake. This time it costed you time to debug it. Sorry :(